src/api/config.rs

```diff
@@ -28,7 +28,7 @@ const MAX_RDO_LOOKAHEAD_FRAMES: usize = usize::max_value() - 1;
 const MAX_MAX_KEY_FRAME_INTERVAL: u64 = i32::max_value() as u64 / 3;
 
 /// Encoder settings which impact the produced bitstream.
-#[derive(Clone, Debug)]
+#[derive(Clone, Copy, Debug)]
```

Comment:
Why do you need to have it Copy? It is fairly large.

Reply:
Because fields get moved when writing the time scale from the config instead of the video details.